### ISSUE TITLE
Question about the classification loss?

### ISSUE THREAD

**Issue by Ha0Tang** [yunjey/stargan#105]
Thanks for your code. I want to know why you use binary_cross_entropy_with_logits for CelebA and use cross_entropy for RaFD.
